src/hooks/useVotingPowerInformation.ts

```diff
@@ -1,20 +1,20 @@
-import useDelegatedVotingPower from './useDelegatedVotingPower'
 import useDelegation from './useDelegation'
 import useVotingPowerBalance from './useVotingPowerBalance'
+import useVotingPowerBalanceList from './useVotingPowerBalanceList'
 
 export default function useVotingPowerInformation(address?: string | null) {
   const { votingPower, isLoadingVotingPower } = useVotingPowerBalance(address)
   const [delegation, delegationState] = useDelegation(address)
-  const { scores, isLoadingScores, delegatedVotingPower } = useDelegatedVotingPower(delegation.delegatedFrom)
+  const { votingPower: scores, isLoadingVotingPower: isLoadingScores } = useVotingPowerBalanceList(delegation.delegatedFrom.map(d => d.delegator))
 
   return {
-    votingPower,
+    votingPower: votingPower?.totalVp || 0,
```

Comment:
if scores is a `Record<string, ScoreDetail>`, shouldn't we use votingPower as 
`votingPower[delegatorAddress].totalVp`?